lib/moonshot/artifact_repository/s3_bucket_via_github_releases.rb

```diff
@@ -115,22 +116,24 @@ def upload_to_s3(file, key)
     # @raise [RuntimeError] If the file fails to download correctly after 3
     #                       attempts.
     def download_from_github(version)
+      file = nil
```

Comment:
I could be wrong, but I think `Retriable.retriable` will return the result of the block, so you could omit this line, and move the `file` at the end of this method to the last line of the block. I suggest giving this a try to verify.